backend/spell.py

```diff
@@ -6,9 +6,42 @@
 from backend.location import find_adjacent_hexes
 from backend.operation import Operation
 from graphics.screen_input import choose_from_list
+import graphics.screen_input as screen_input
 from copy import deepcopy
 import backend.location as location
 
+""" 
+method to place auras on hexes, used in Imposter and Upset
+place_auras_on_hexes replaces the auras on the hexes in hex_list with 
+those in aura_list, based on user input
+
+Params:
+ - aura_list: list of auras to place (entries should be 'Dark' or 'Light'). None entries
+    will automatically be removed
+ - hex_list: list of hexes to put auras on
+"""
+def place_auras_on_hexes(board, aura_list, hex_list):
```

Comment:
any reason to take `aura_list` as a param vs calculating it here based on `hex_list`?

Reply:
For the Imposter, one needs a different list of auras than those on hex_list, namely, the list of auras in the Imposter's room